packages/react-composites/src/composites/CallWithChatComposite/styles/EmbeddedPeoplePane.styles.ts

```diff
@@ -1,16 +1,38 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { IButtonStyles } from '@fluentui/react';
+import { IButtonStyles, IStackStyles, IStackTokens } from '@fluentui/react';
+
+/**
+ * @private
+ */
+export const peoplePaneContainerTokens: IStackTokens = {
+  childrenGap: '0.5rem'
+};
+
+/**
+ * @private
+ */
+export const localAndRemotePIPContainerStyles: IStackStyles = { root: { height: '100%' } };
```

Comment:
I dont understand how the PiPiP is overlaid on top of other content (where is the zIndex set? How large is the element? Does it cover the entire screen, but let's clicks go through?)

It would be useful to add a comment here explaining what area on the screen this container fills (and so, why is 100% needed).

Also, could you instead set `grow` prop in the `Stack`?

Reply:
Yes, the `grow` prop works. Removed that `localAndRemotePIPContainerStyles`. The PiPiP is not overlaid in this PR. It is just a child of the Stack. The Stack items are the PartcipantList, PiPiP, and the 'Copy Invite Link' button, in that order. Working on making it overlaid and draggable.